8.0.Dockerfile

```diff
@@ -29,6 +29,17 @@ ONBUILD ARG PGUSER=odoo
 ONBUILD ARG PGPASSWORD=odoopassword
 ONBUILD ARG PGHOST=db
 ONBUILD ARG PGDATABASE=prod
+ONBUILD ARG ODOO_MAX_CRON_THREADS=2
```

Comment:
I think a better default is 1. 2 crons can collide each other and produce more concurrency errors, and besides cron jobs are usually not that important that can't be delayed a little bit until the previous job finishes. It's better to default to 1 and let customizations increase that number.